flex/03-flex-header-2/style.css

```
@@ -45,4 +48,18 @@ a {
 
 ul {
   list-style-type: none;
+  display: flex;
+  padding: 0px;
+  margin-left: 10px;
+  gap: 15px;
 }
+
+.container-left,
+.container-right {
+  display: flex;
+  align-items: center;
+}
+
+.container-right div {
```

```suggestion
.container-right .<class-of-the-inner-div> {
```
Avoid putting HTML tags as selectors in CSS. Give that div a class.